lab-anthony/gulpfile.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
+const nodemon = require('gulp-nodemon');
+
+const paths = ['*.js', 'lib/*.js', 'model/*.js', 'route/*.js', 'test/*.js'];
+
+gulp.task('lint', function() {
+  gulp.src(paths)
+    .pipe(eslint())
+    .pipe(eslint.format());
+});
+
+gulp.task('test', function(){
```

Comment:
One thing that might be nice here would be to write a task to empty out the saved notes you have for cleanliness.